submissions/ik-web/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,280 @@
+<!Doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Movie Seat Booking</title>
+    <link rel="icon" href="img/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <main class="main">
+        <section class="movie-info">
+            <h1 class="movie-info__title">Monster Hunter</h1>
+            <div class="movie-info__inner">
+                <div class="movie-info__poster">
+                    <img src="img/movie.jpg" alt="movie poster">
+                </div>
+
+                <ul class="movie-info__about">
+                    <li class="movie-info__about-item about-item__format">3D</li>
+                    <li class="movie-info__about-item about-item__map">Kyiv, SEC "Lavina", Hall 13</li>
```

Comment:
For address use HTML tag `addres` [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address)

Reply:
I fixed this point, thanks for the comment!